src/db/migration/zmdbupgrade.pl

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/perl
+use strict;
+use lib "/opt/zimbra/libexec";
+use lib "/opt/zimbra/libexec/scripts";
+use lib "/opt/zimbra/common/lib/perl5";
+use Migrate;
+use zmupgrade;
+
+my $platform = qx(/opt/zimbra/libexec/get_plat_tag.sh);
+chomp $platform;
+my $su = "su - zimbra -c";
+my $hiVersion = 118;
```

Comment:
@umagmrit This needs documented for updating the version in this script too when there is DB update being made in future.

Reply:
@dasiyogesh updated same in the https://github.com/Zimbra/zm-build/pull/278/files#diff-c0504a8daeec38ee744cda949abc3efa31ea1cc54a4579905856bf49613277a4R45